exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if ( mark >= 80){
+    return "A";
+  } else if (mark >= 60){
+    return "B";
+  } else if (mark >= 50) {
+    return "C";
+  } else {
+    return "F";
+  }
+}
```

Comment:
hi, I'm not sure here but defining the intervals with starting and ending numbers would be better not to cause an error. 
ex:
   else if(mark < 80 && mark > 60) {
    return "B";

Reply:
I agree, I used this method because it saves time.

Reply:
This code is correct, you don't need to specify the range since in the line 14 for example the mark surely will be smaller than 80, otherwise it would have returned from the line 13.